custom/icds_reports/reports/awc_reports.py

```diff
@@ -851,33 +874,38 @@ def get_data_for(query_class, filters):
                     'all': get_value(data, 'css_lactating_all'),
                     'format': 'percent_and_div',
                     'frequency': frequency
-                },
-                {
-                    'label': _(ADOLESCENT_GIRLS_ENROLLED_FOR_ANGANWADI_SERVICES),
-                    'help_text': _((
-                        "Of the total number of adolescent girls (aged 11-14 years), the percentage "
-                        "of girls enrolled for Anganwadi Services"
-                    )),
-                    'percent': percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    ),
-                    'color': get_color_with_green_positive(percent_diff(
-                        'person_adolescent',
-                        data,
-                        prev_data,
-                        'person_adolescent_all'
-                    )),
-                    'value': get_value(data, 'person_adolescent'),
-                    'all': get_value(data, 'person_adolescent_all'),
-                    'format': 'percent_and_div',
-                    'frequency': frequency
                 }
             ]
         ]
     }
+    if (not beta) or selected_month.date() >= ADOLESCENT_GIRLS_DATA_THRESHOLD:
+        adolescent_girls = {
```

Comment:
this might be easier to read with separate sections with `if beta` and `else`.

Reply:
Similarly, [here](https://github.com/dimagi/commcare-hq/pull/26293/commits/55ac0e4e4d69c7c06ffa21a60fac8ea39cfa8876#diff-a193d2965b97f074a335d160cf05beaaR189)